README.md

```diff
@@ -84,3 +84,51 @@ For example, if we wanted to exclude the copyright/license check for a directory
   }
 }
 ```
+
+# Creating a Whitelist for Copyright/License Check in Repolinter
+  * To include only specific directories (e.g., `/test1-data`, `/test2-data`) in the copyright/license check using Repolinter, follow these steps:
+
+For example, if we wanted to create a whiltelist the copyright/license check for list of directory (e.g. `/test1-data`, `/test2-data`) from Repolinter:
```

Comment:
This line is repetitive given the previous line above it. Can we get rid of one of them?